### PR TITLE
[ENH] author and maintainer tags for forecasting module

### PR DESCRIPTION
**Pull request by fkiraly**
Adds author and maintainer tags to all estimators in the forecasting module - based on best knowledge where documentation was missing.

### CODE REVIEW

File: sktime/forecasting/varmax.py [hunk 1]
```diff
@@ -206,6 +206,12 @@ class VARMAX(_StatsModelsAdapter):
     """
 
     _tags = {
+        # packaging info
+        # --------------
+        "author": "KatieBuc",
```
Comment (yarnabrina) [inline sktime/forecasting/varmax.py:211]:
```suggestion
        "authors": "KatieBuc",
```

**Comment by fkiraly** [comment]
> but this must be an extremely tedious task to do this for every single estimator and finding all authours from git history. Can't appreciate that enough.

Not as bad as it looks with judicious application of copy-paste and AI assistants - but still tedious of course, the AI keeps inventing tags and contributors...

File: sktime/forecasting/adapters/_hcrystalball.py [hunk 2]
```diff
@@ -106,10 +106,16 @@ class HCrystalBallAdapter(BaseForecaster):
     """
 
     _tags = {
+        # packaging info
+        # --------------
+        "authors": "MichalChromcak",
+        "maintainers": "MichalChromcak",
```
[the remark this answers is not in the document]
Comment (fkiraly) [reply sktime/forecasting/adapters/_hcrystalball.py:112]: The root specification is `("list", "str") - any individual string or list of strings is valid` (line 23 of `_tags.py`), so either is fine

File: sktime/forecasting/varmax.py (hunk 1 above)
Comment (fkiraly) [reply sktime/forecasting/varmax.py]: yes, found it through tests already

File: sktime/forecasting/compose/_pipeline.py [hunk 3]
```diff
@@ -1707,6 +1710,7 @@ class Permute(_DelegatedForecaster, BaseForecaster, _HeterogenousMetaEstimator):
     """
 
     _tags = {
+        "author": "aiwalter",
```
[the remark this answers is not in the document]
Comment (fkiraly) [reply sktime/forecasting/compose/_pipeline.py]: yes, found it through tests already

**Pull request merged**